service/src/test/java/bio/terra/workspace/service/resource/ResourceLineageUtilsTest.java

```diff
@@ -0,0 +1,127 @@
+package bio.terra.workspace.service.resource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import bio.terra.workspace.common.BaseUnitTest;
+import bio.terra.workspace.db.DbSerDes;
+import bio.terra.workspace.db.model.DbResource;
+import bio.terra.workspace.service.resource.model.CloningInstructions;
+import bio.terra.workspace.service.resource.model.ResourceLineageEntry;
+import bio.terra.workspace.service.resource.model.StewardshipType;
+import bio.terra.workspace.service.resource.model.WsmResourceFamily;
+import bio.terra.workspace.service.resource.model.WsmResourceType;
+import bio.terra.workspace.service.resource.referenced.cloud.any.datareposnapshot.ReferencedDataRepoSnapshotAttributes;
+import bio.terra.workspace.service.resource.referenced.cloud.any.datareposnapshot.ReferencedDataRepoSnapshotResource;
+import bio.terra.workspace.service.workspace.model.CloudPlatform;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class ResourceLineageUtilsTest extends BaseUnitTest {
+
+  @Test
+  public void constructResourceNullLineage_resourceLineageEmptyArray() {
+    UUID randomId = UUID.randomUUID();
+    String resourceName = "testdatarepo-" + randomId;
+    Map<String, String> propertyMap = new HashMap<>();
+
+    var resource =
+        new ReferencedDataRepoSnapshotResource(
+            randomId,
+            randomId,
+            resourceName,
+            "description of " + resourceName,
+            CloningInstructions.COPY_NOTHING,
+            "terra",
+            "polaroid",
+            /*resourceLineage=*/ null,
+            propertyMap);
+
+    assertEquals(resource.getResourceLineage(), new ArrayList<>());
```

Comment:
Technically this should be flipped

assertEquals(new ArrayList<>(), resource.getResourceLineage());